client/src/Movies/MovieList.js

```diff
@@ -34,7 +35,9 @@ export default class MovieList extends Component {
 function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
+    <Link to={'/movie/${movie.id}'}>
```

Comment:
Consider using back-ticks in here for string literals, you are using single quotes instead. With this fixed it will dynamically change the string properly.